met-web/src/routes/AuthenticatedRoutes.tsx

```diff
@@ -79,34 +81,39 @@ const AuthenticatedRoutes = () => {
                         path="wizard"
                         handle={{ crumb: () => ({ name: 'New Engagement' }) }}
                         element={<EngagementCreationWizard />}
-                        loader={languageLoader}
                     />
                 </Route>
                 <Route
                     path=":engagementId"
                     id="single-engagement"
                     errorElement={<NotFound />}
                     loader={engagementLoader}
+                    handle={{
```

Comment:
Thanks for adding this other crumb. Do you know why we have both this route and "view" that point to the view page?